### PR TITLE
[DE] Translations for PR #703 over in the lubelog Repo

### PR DESCRIPTION
**Pull request by Scorpoon**
[PR #703](https://github.com/hargata/lubelog/pull/703)

Should be merged only when the PR is also accepted

### CODE REVIEW

File: Europe/de.json [hunk 1]
```diff
@@ -352,5 +352,9 @@
     "Visible_Tabs": "Sichtbare Registerkarten",
     "Whichever_comes_first": "Was zuerst eintritt",
     "Year": "Jahr",
-    "Year(must_be_after_1900)": "Jahr (muss nach 1900 sein)"
+    "Year(must_be_after_1900)": "Jahr (muss nach 1900 sein)",
+    "No_Vehicles_Available": "Keine Fahrzeuge verfügbar",
+    "Use_this_tool_to_migrate_data_between_LiteDB_and_Postgres": "Dieses Tool dient der Datenmigration zwischen LiteDB and Postgres",
```
Comment (steffenrapp) [inline Europe/de.json:357]:
```suggestion
    "Use_this_tool_to_migrate_data_between_LiteDB_and_Postgres": "Dieses Tool dient der Datenmigration zwischen LiteDB und Postgres",

```

File: Europe/de.json [hunk 2]
```diff
@@ -352,5 +352,9 @@
     "Visible_Tabs": "Sichtbare Registerkarten",
     "Whichever_comes_first": "Was zuerst eintritt",
     "Year": "Jahr",
-    "Year(must_be_after_1900)": "Jahr (muss nach 1900 sein)"
+    "Year(must_be_after_1900)": "Jahr (muss nach 1900 sein)",
+    "No_Vehicles_Available": "Keine Fahrzeuge verfügbar",
+    "Use_this_tool_to_migrate_data_between_LiteDB_and_Postgres": "Dieses Tool dient der Datenmigration zwischen LiteDB and Postgres",
+    "Note_that_it_is_recommended_that_the_Postgres_DB_is_empty_when_importing_from_LiteDB_to_prevent_primary_key_errors": "Für den Import der LiteDB ist es empfohlen eine leere Postgres DB zu verwenden, um Primärschlüsselfehler zu vermeiden.",
```
Comment (steffenrapp) [inline Europe/de.json:358]:
```suggestion
    "Note_that_it_is_recommended_that_the_Postgres_DB_is_empty_when_importing_from_LiteDB_to_prevent_primary_key_errors": "Für den Import der LiteDB wird empfohlen, eine leere Postgres DB zu verwenden, um Primärschlüsselfehler zu vermeiden.",

```

**Review by steffenrapp — CHANGES_REQUESTED** [review]
Thank you for the update :) I made 2 comments.

**Comment by Scorpoon** [comment]
> Thank you for the update :) I made 2 comments.

Thanks for your suggestions, I have adopted them.

**Review by steffenrapp — APPROVED** [review]

**Pull request merged**